freqtrade/persistence/models.py

```diff
@@ -359,7 +359,14 @@ def set_isolated_liq(self, isolated_liq: Optional[float]):
                 is_short=self.is_short,
                 leverage=self.leverage,
                 trading_mode=self.trading_mode,
-                collateral=Collateral.ISOLATED
+                collateral=Collateral.ISOLATED,
+                mm_ex_1=0.0,
+                upnl_ex_1=0.0,
+                # TODO-lev: position=amount * current_price,
+                # TODO-lev: maintenance_amt,
+                # TODO-lev: wallet_balance,
+                # TODO-lev: mm_rate,
```

Comment:
We'll need to figure out how to get these values to pass in

Reply:
Another option is that we can calculate this value inside freqtradebot and assign trade.set_isolated_liq(value)

Reply:
i might not be understanding the question?

Reply:
> i might not be understanding the question?
> after a quick glance - it seems like the only thing that's not available as part of the trade is `wallet_balance` ... which we can (and should) pass in for sure.

ok, nevermind

Reply:
The only two missing params left here are `maintenance_amt`, `mm_rate`
Which I don't know how to get or calculate